src/lib/components/error.svelte

```diff
@@ -25,19 +26,19 @@
   <h1 class="text-[12rem] font-semibold ">{status}</h1>
   <p class="-mt-6 mb-5 text-lg">Uh oh. There's an error.</p>
   <p class="text-lg">
-    <a
+    <Link
       href={currentLocation}
       on:click={() => {
         if (browser) {
           window.location.reload();
         }
       }}
       target="_self"
-      class="underline-offset-2 underline">Try a refresh</a
+      class="underline-offset-2 underline">Try a refresh</Link
```

Comment:
You could probably drop some of the underline classes here because they're defined in the component.

Reply:
Oh, good callout.